src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/CommonStepDef.scala

```diff
@@ -20,42 +20,10 @@ import uk.gov.hmrc.test.ui.cucumber.stepdefs.CommonStepDef.genSequenceId
 import uk.gov.hmrc.test.ui.pages.base.CommonPage.{continue, continueOnMiniCya}
 import uk.gov.hmrc.test.ui.pages.base.Constants.yes
 import uk.gov.hmrc.test.ui.pages.base.{CommonPage, Constants}
-import uk.gov.hmrc.test.ui.pages.section1.{
-  ChoicePage,
-  DeclarantDetailsPage,
-  DeclarationTypePage,
-  DoYouHaveADucrPage,
-  DucrEntryPage,
-  EnterAMucrPage,
-  LinkMucrPage,
-  LoginPage,
-  LrnPage,
-  StandardOrOtherPage,
-  SummarySection1Page
-}
-import uk.gov.hmrc.test.ui.pages.section2.{
-  AreYouTheExporterPage,
-  AuthorisationPage,
-  AuthorisationsListPage,
-  AuthorisationsYesNoPage,
-  CarrierAddressPage,
-  CarrierEORINumberPage,
-  ConsigneeDetailsPage,
-  ExporterAddressPage,
-  ExporterEORINumberPage,
-  OnBehalfOfAnotherAgentPage,
-  OtherPartiesInvolvedListPage,
-  OtherPartiesInvolvedPage,
-  ProcedureChoicePage,
-  RepresentationTypeAgreedPage,
-  RepresentativesEORINumberPage
-}
-import uk.gov.hmrc.test.ui.pages.section3.{
-  CountryOfRoutingPage,
-  DestinationCountryPage,
-  LocationOfGoodsPage,
-  OfficeOfExitPage
-}
+import uk.gov.hmrc.test.ui.pages.section1.{ChoicePage, DeclarantDetailsPage, DeclarationTypePage, DoYouHaveADucrPage, DucrEntryPage, EnterAMucrPage, LinkMucrPage, LoginPage, LrnPage, StandardOrOtherPage, SummarySection1Page}
```

Comment:
In order to shorten these import I would use the underscore.
e.g. `import uk.gov.hmrc.test.ui.pages.section1._`

Same for the `import`s below